### PR TITLE
Add CLX support

### PR DESCRIPTION
**Pull request by glebm**
CLX is identical to CL2 except for the frame header, making it easy to maintain support for it.

### CODE REVIEW

File: source/d1clx.cpp [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#include "d1clx.h"
+
+quint16 D1ClxFrame::computeWidthFromHeader(QByteArray &rawFrameData)
+{
+    QDataStream in(rawFrameData);
+    in.setByteOrder(QDataStream::LittleEndian);
+    in.skipRawData(2);
+    quint16 result;
+    in >> result;
+    in.skipRawData(6);
```
Comment (AJenbo) [inline source/d1clx.cpp:10]: nit: is this needed

**Review by AJenbo — APPROVED** [review]

**Comment by AJenbo** [comment]
Thanks this solves https://github.com/savagesteel/d1-graphics-tool/issues/16

File: source/d1clx.cpp (hunk 1 above)
Comment (glebm) [reply source/d1clx.cpp:10]: Yes, it is expected that the input is past the header after calling this function

Comment (AJenbo) [reply source/d1clx.cpp:10]: Oh, I didn't think the pointer survived past the function call since `QDataStream` was created inside of it.

**Pull request merged**